apps/examples/testcase/le_tc/network/Kconfig

```diff
@@ -49,6 +50,10 @@ config TC_NET_SOCKET
 	bool "socket() api"
 	default n
 
+config TC_NET_PKT
+	bool "pubf() api "
```

Comment:
maybe `pbuf`

Reply:
let's keep it same. for other config values it is like above.
e.g. config TC_NET_SETSOCKOPT
        bool "setsockopt() api"
        default n

config TC_NET_CONNECT
        bool "connect() api"
        default n

Reply:
I mean typo. pubf is a typo of pbuf. No?

Reply:
ohh.. will correct that.